frontend/ui/size.lua

```diff
@@ -27,6 +27,17 @@ defaults, please take a second to consider:
 
 local dbg = require("dbg")
 local Screen = require("device").screen
+local Device = require("device")
```

Comment:
Other way around, both for alphabetic and semantic reasons, and then `local Screen = Device.screen` ;).

Reply:
Gotcha! Will fix

Reply:
Fixed! On the other hand, a quick search in files shows there are other 41 cases of `require("device").screen`...

Reply:
Because most of those don't need `Device` itself, just `Screen` ;).